tests/rptest/tests/enterprise_features_license_test.py

```diff
@@ -42,12 +43,26 @@ class Features(IntEnum):
     leadership_pinning = 10
 
 
+# TODO(oren): extend to multiple configs?
```

Comment:
Do you mean a test for a partially successful config update? Agree that that would be nice to have.

Reply:
yeah partially successful patch, or trying to switch on multiple enterprise features. doesn't seem worth all the juggling.

Reply:
removed